app/models/split_creation.rb

```diff
@@ -21,7 +21,7 @@ def self.create(params)
   end
 
   def weighting_registry=(registry)
-    @weighting_registry = registry.each_with_object({}) { |(variant_name, weight), r| r[variant_name.to_s] = weight }
+    @weighting_registry = registry.to_h.stringify_keys
```

Comment:
Is this doing anything more than just stringifying keys? I already added the `to_h` because ActionController::Parameters is getting slimmed down (won't inherit from Hash).

Reply:
yeah, seems correct.